recipes/_start.rb

```diff
@@ -5,11 +5,38 @@
 #
 # Ensures the splunk instance is running, and performs post-start tasks.
 
+def restart_needed?(ulimit_command)
```

Comment:
minor, thoughts on just doing this inline rather than putting it in a function?
````
ulimit_command = "..."
init_file_path = '...'
restart_needed = !File.exist(init_file_path) || File.readlines(...).none?
````
Then the file_path could be reused down on line 27.